kuma/wiki/tests/conftest.py

```diff
@@ -185,6 +185,103 @@ def doc_hierarchy(wiki_user):
 
 
 @pytest.fixture
+def doc_hierarchy_with_zones(settings, doc_hierarchy, wiki_user_2, wiki_user_3):
```

Comment:
Yeah, I've been putting most things if not everything into the higher-level ``conftest.py`` files, thinking they would most likely be used by more than one test file, but I agree that we should eventually (or even now if you prefer) put test-specific fixtures in the test files.

In this case, ``doc_hierarchy_with_zones`` is actually used by multiple files, so I think it belongs here, but ``doc_hierarchy`` is not used (except by ``doc_hierarchy_with_zones``) and ``root_doc_with_mixed_contributors`` is only used in ``test_jobs.py``. I'm going to fold ``doc_hierarchy`` into ``doc_hierarchy_with_zones`` since it's not used by any tests.